cli/src/stake.rs

```
@@ -567,6 +591,37 @@ pub fn process_show_stake_account(
     }
 }
 
+pub fn process_show_stake_history(
+    rpc_client: &RpcClient,
+    _config: &CliConfig,
+    use_lamports_unit: bool,
+) -> ProcessResult {
+    let stake_history_account = rpc_client.get_account(&stake_history::id())?;
+    let stake_history = StakeHistory::from_account(&stake_history_account).unwrap();
```

you can throw an error here